torch/csrc/jit/runtime/static/ops.cpp

```diff
@@ -340,7 +340,7 @@ REGISTER_OPERATOR_FUNCTOR(aten::clamp, aten_clamp, [](Node* n) -> SROperator {
     } else {
       auto in1_s = p_node->Input(1).toOptional<at::Scalar>();
       auto in2_s = p_node->Input(2).toOptional<at::Scalar>();
-      at::native::clamp_out(in0_t, in1_s, in2_s, out_t);
+      out_t = at::clamp(in0_t, in1_s, in2_s);
```

Comment:
This cannot be landed as is. How do I call the native version of the kernel I ported?

Reply:
If I understand your question correctly, I think just `out_t = at::clamp_out(in0_t, in1_s, in2_s, out_t)` will work.

Compare with [thnn_conv2d_out](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L9387) which can be called `at::thnn_conv2d_out(out, x, weight, {1,1}, nullopt)`.

Reply:
Won't that still involve the dispatcher? Static runtime is supposed to not do that, right?

Reply:
Sorry you're right. Hmm I guess one hacky solution would be

```cpp
    c10::optional<at::Tensor> in1_t;
    c10::optional<at::Tensor> in2_t;
    if (p_node->Input(1).isTensor()) {
      in1_t = p_node->Input(1).toOptional<at::Tensor>();
      in2_t = p_node->Input(2).toOptional<at::Tensor>();
    } else {
      in1_t = c10::make_optional(p_node->Input(1).toTensor());
      in2_t = c10::make_optional(p_node->Input(2).toTensor());
    }
    at::native::clamp_out(in0_t, in1_t, in2_t, out_t);
```

Reply:
Don't do that: `at::cpu::clamp_out` is the name of the game. There are other occurrences of this pattern in this file too.